### ISSUE TITLE
[Suggestion]: Enable keepSkills gamerule in University 1.16 by default

### ISSUE THREAD

**Issue by aaronhowser1** [FTBTeam/FTB-Modpack-Issues#429]
### Modpack

FTB University 1.16

### Is your suggestion related to a problem? Please describe.

You lose EFM skills on death, which is lame. Need to find a mod to allow default gamerules.

**Comment by Raidobw2** [comment]
Or use built-in function load with datapacks since KubeJS loads one.

Example structure and contents in `kubejs/data`:
Have `minecraft/tags/function/load.json` as silently stated in https://minecraft.fandom.com/wiki/Tag#Function_tags.

`load.json` will contain a very simple structure with resource location:
```
{
	"values": [ 
		"gamerules:load"
	]
}
```
Namespace "gamerules" in that example and filename "load" (Can rename both, but "functions" folder gotta be in the middle).
Have that be in `kubejs/data` as well then, as according to resource location:
`gamerules/functions/load.mcfunction`.
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; ^ Functions folder, hardcoded name

Populate e.g. `load.mcfunction` with commands you want to run on world load, example:
```
gamerule doDaylightCycle false
time set midnight
```
P.S. Commands get ran everytime a world is joined, every reload, etc.
Example of that being applied and the result for comprehension:
![world_creation](https://user-images.githubusercontent.com/17010632/145000898-24d8d038-1297-43d4-ba55-2fecd017b6cb.gif)
Cool stuff :D

Ah yeah, forgot to mention cause I'm dumbo and used creative which has cheats (my mind is programmed to click creative woops) -> You don't need cheats for that. It's a datapack, the commands are not ran by the player. Do SP for your tests you'll see!

**Comment by aaronhowser1** [comment]
So I can have this file:
![image](https://user-images.githubusercontent.com/15698722/146795018-bf090fc6-e97a-4bc9-8bb8-ae92e0ef9e63.png)
with the contents
```json
{
	"values": [ 
		"ftbuniversity:on_world_load"
	]
}
```
and then have this other file:
![image](https://user-images.githubusercontent.com/15698722/146795186-9c746684-5122-4342-a74b-be55299e1281.png)
with the contents
```
gamerule keepSkills true
```
and it will enable that gamerule every time the world is loaded?

Just tested, it did not work :/

**Comment by Raidobw2** [comment]
@aaronhowser1 I made a typo woops. That should work, I'll fix my original message
![image](https://user-images.githubusercontent.com/17010632/146919352-9fffecfa-36fc-4647-9c79-4ce4db42635c.png)
The "functions" folder always takes a "S" like functionS, I just checked back on my test instance, here's my paths to confirm:
`kubejs\data\minecraft\tags\functions\load.json`
`kubejs\data\gamerules\functions\load.mcfunction`
Your file contents are fine, I think it's just that first path where I messed up. Add a S. Lmk if that made it work, and yeah everytime the world is loaded.
